components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/resources/admin-api.yaml

```diff
@@ -2248,6 +2248,127 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+######################################################
+# The BotDetectedData API resources
+######################################################
+
+  #-----------------------------------------------------
+  # Get configured email list
+  #-----------------------------------------------------
+  /botData/getEmailList:
```

Comment:
This is not RESTful. We need to redesign this part. 

Something like this `/bot-detection/emails`. Please create a GIT issue to track this.